sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -53,13 +51,21 @@ def __init__(self, context):
         super().__init__(context)
 
         self.context = context
+        self.init_ui()
 
-        self._hand_finder = HandFinder()
-        self._hand_parameters = self._hand_finder.get_hand_parameters()
-        self.joint_prefix = next(iter(list(self._hand_parameters.joint_prefix.values())))
-        self.hand_joints = self._hand_finder.get_hand_joints()
+    def _detect_hand_id_and_joints(self):
```

Comment:
I am not sure if it was the case before, but would this work if two hands are connected?

Reply:
The plugin is supposed to work with one hand at time, in case of two hands connected it will assign the id of the first detected hand from the joint_states (in the new implementation)